### PR TITLE
[zh] Add Chinese docs

### PR DESCRIPTION
**Pull request by Zhanxueyou**
Signed-off-by: zhanxueyou <zhanxueyou@msn.com>

### CODE REVIEW

File: content/zh/docs/chart_template_guide/wrapping_up.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+title: "下一步"
+description: "收尾 - 一些能帮到你的指向其他的文档的链接。"
+weight: 14
+---
+
+本指南旨在为chart开发者提供对如何使用Helm模板语言的强大理解能力。该模板聚焦于模板开发的技术层面。
+
+但涉及到chart的实际日常开发时，很多内容本指南并没有覆盖到。这里有一些有用的文档链接帮助你创建新的chart:
+
+- [Helm的Chart项目](https://github.com/helm/charts) 是不可或缺的chart资源。这个项目也定义了
```
Comment (bridgetkromhout) [inline content/zh/docs/chart_template_guide/wrapping_up.md:11]: We recently updated this to point to Artifact Hub: https://github.com/helm/helm-www/pull/1016

File: content/zh/docs/topics/chart_tests.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+这是一个helm对[bitnami wordpress chart](https://hub.helm.sh/charts/bitnami/wordpress)的pod定义的测试。
+如果你下载了一个chart的拷贝，可以在本地看到以下文件：
+
+```console
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm pull bitnami/wordpress --untar
+```
+
+```consol
```
Comment (bridgetkromhout) [inline content/zh/docs/topics/chart_tests.md:37]: I think this should be ```console with an `e`?

**Review by bridgetkromhout — APPROVED** [review]
Thank you! I commented with a few small items you may want to fix, but I am going to go ahead and merge this now, and you can fix those in a future PR. Thanks!

File: content/zh/docs/chart_template_guide/wrapping_up.md (hunk 1 above)
Comment (Zhanxueyou) [reply content/zh/docs/chart_template_guide/wrapping_up.md:11]: Got it, I'll update this point.

**Pull request merged**